soc/nxp/s32/s32k3/Kconfig.defconfig

```diff
@@ -28,4 +28,9 @@ endif # ETH_DRIVER && NET_L2_ETHERNET
 config CACHE_MANAGEMENT
 	default y
 
+# S32K3 places the vector table at +0x400
+# This makes the bootloader and the sign tool compatible with that.
+config ROM_START_OFFSET
+	default 0x400 if (BOOTLOADER_MCUBOOT || MCUBOOT)
```

Comment:
then this would be for `BOOTLOADER_MCUBOOT` only, otherwise it will shift MCUboot itself by 0x400 and never boot it

Reply:
@nordicjm  My MCUboot Vector Table is also at 0x400 offset (0x402400)
IVT's  4th word also points to 0x00402400

Reply:
So what is the first 0x400? Why doesn't the flash controller point to the start of usable flash?

Reply:
@nordicjm .. Corrected.. please check now